src/main/java/edu/harvard/iq/dataverse/Dataverse.java

```diff
@@ -105,7 +106,7 @@ public void setDataverseType(DataverseType dataverseType) {
     private final String uncategorizedString = "Uncategorized";
 
     /**
-     * @todo Don't hard code these as English.
+     * @todo Don't hard code these as English. Use bundleUtils
```

Comment:
I can do that. I can actually change all of them if need be, as they all are given in the bundle. Should I do it in this branch only?